packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -295,8 +303,11 @@ export default function PostOptionsMenu({
       ),
       label: `${post?.bookmarked ? 'Remove from' : 'Save to'} bookmarks`,
       action: onToggleBookmark,
-    },
-    {
+    });
+  }
+
+  if (shouldUseFeedLayoutV1) {
```

Comment:
shouldn't this be `!shouldUseFeedLayoutV1` ?

Reply:
Oh, right. Will update quickly.